core/uploadDiveShare.cpp

```diff
@@ -45,13 +45,19 @@ void uploadDiveShare::doUpload(bool selected, const QString &uid, bool noPublic)
 	reply = manager()->put(request, json_data);
 
 	// connect signals from upload process
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+	connect(reply, &QNetworkReply::finished, this, &uploadDiveShare::uploadFinishedSlot);
+	connect(reply, &QNetworkReply::errorOccurred, this, &uploadDiveShare::uploadErrorSlot);
```

Comment:
Small rant: Jeez. I find that function renaming so tiresome. There are so many API problems that could/should be improved, why waste time on stupid function names?

Reply:
yes... my jaw dropped with how many pointless renaming exercises they went through.
Sure the new names are "slightly more logical and consistent", but what the actual FART. Such a waste of time for the users.